core/locale/main/values/strings_ui.xml

```diff
@@ -308,11 +309,19 @@
     <string name="xapi_options_what">What</string>
     <string name="xapi_options_when">When</string>
     <string name="xapi_options_where">Where</string>
+    <string name="xapi_options_content">Content</string>
+
+    <string name="report">Report</string>
+    <string name="verb">Verb</string>
+    <string name="id_verbentity_urlid">ID: %s</string>
+    <string name="verb_filter">verb filter</string>
```

Comment:
Could we have a string substitution instead? E.g. 
```
<string name="x_filter">%1$s filter</string>
```
Then that string can be used to create Person filter, content filter, etc. That would reduce the number of words that are part of the translation.